src/Slices/Status/UI/StatusItem.tsx

```diff
@@ -13,11 +13,21 @@ import styled from "styled-components";
 
 interface Props {
   name: string;
-  details: [string, string][];
+  details: [string, Record<string, string> | string][];
```

Comment:
I don't know for you, but I find this is a bit hard to read.
I'd personaly go for something a bit more explicit like this:

```TS
type DetailKey = string;
type DetailValue = Record<string, string> | string;
type DetailTuple = [DetailKey, DetailValue];
```

And then use the DetailTuple type here and in other places where this is needed